autofill/autofill-api/src/main/java/com/duckduckgo/autofill/api/store/AutofillStore.kt

```diff
@@ -87,6 +87,11 @@ interface AutofillStore {
      */
     suspend fun getAllCredentials(): Flow<List<LoginCredentials>>
 
+    /**
+     * Returns all credentials that have been modified since the given timestamp
+     */
+    suspend fun getAllCredentialsModifiedSince(modifiedSince: Long): List<LoginCredentials>
```

Comment:
I'm assuming this `modifiedSince` is milliseconds since 1970, but would be good to make that more explicit here (e.g, add javadoc for the parameter)

Reply:
This is now removed as I'm moving away from using AutofillStore (as discussed)